decidim-elections/app/assets/config/decidim_votings_manifest.js

```diff
@@ -0,0 +1 @@
+//= link_tree ../images/decidim
```

Comment:
Just in case, do we need to add this to the list of files to compile on the production env?

Reply:
This is needed if the marker popup has an icon, but we are not using one, so I'll remove this line.